api_app/filters.py

```diff
@@ -55,6 +56,14 @@ def filter_for_name(queryset, value, name, *args, **kwargs):
             Q(observable_name__icontains=name) | Q(file_name__icontains=name)
         )
 
+    @staticmethod
+    def filter_for_playbook_to_execute(
+        queryset, value, playbook_to_execute, *args, **kwargs
+    ):
+        if playbook_to_execute.lower() == "custom analysis":
+            return queryset.filter(playbook_to_execute=None)
+        return queryset.filter(playbook_to_execute__name__icontains=playbook_to_execute)
+
```

Comment:
Uhm no. Compare of arbitrary strings is not a good practice.
Moreover, if a playbook is called `custom analysis` this will never return the correct playbook.

_I think_ tha you can just use 
```python3
    playbook_to_execute = filters.CharFilter(field_name="playbook_to_execute__name", lookup_expr="icontains")
```
And the library (I hope) will manage the null case itself (if not, keep the `filter_for_playbook_to_execute` but just check that is value is null).